rpc_jobs/lem_multi_node_aio.yml

```diff
@@ -146,14 +146,15 @@
             ) // deploy_sh
             multi_node_aio_prepare.multi_node_aio_networking()
             node(){{
-            maas.prepare(instance_name: instance_name)
-            maas.deploy()
-            maas.verify()
-            tempest.tempest("infra1", "deploy1")
-            kibana.kibana("deploy1")
-            holland.holland()
-          }} // hardware node
-          ssh_slave.destroy()
-          maas.entity_cleanup(instance_name: instance_name)
-        }} // cit node
-      }} // timeout
+              maas.prepare(instance_name: instance_name)
```

Comment:
This job contained a syntax error, so had to be fixed in order to not break the gate with the new lint job.